src/rastervision/command/predict_command.py

```diff
@@ -12,3 +12,14 @@ def run(self, tmp_dir):
         msg = 'Making predictions...'
         click.echo(click.style(msg, fg='green'))
         self.task.predict(self.scenes, tmp_dir)
+
+
+class NoOpPredictCommand(NoOpCommand):
```

Comment:
This seems leftover from the previous changes (along with predict_command_config changes)